### PR TITLE
88 enhancement organisation and style

### PR DESCRIPTION
**Pull request by AKMaily**
I added a Style.hpp, DevicesMenu.cpp and SideBarMenu.cpp. In this files the Style, DevicesMenu and SideBarMenu are created. Did not change the code directly only the structure of the code.

### CODE REVIEW

**Comment by R-Abbasi** [comment]
A few suggestions for this current version:
- Headers are usually expected to use `#include` guards to avoid being `#include`d several times.
- `#include`ing `.cpp` files appear to be against the convention/norm decreasing the readability, I suppose. (What if another developer reads the code facing them?)
- Declaring a function with many parameters seems uncommon - especially when it contains only a few lines of code.

**Review by R-Abbasi — COMMENTED** [review]
A few suggestions for this current version:
- Headers are usually expected to use `#include` guards to avoid being `#include`d several times.
- `#include`ing `.cpp` files appear to be against the convention/norm decreasing the readability, I suppose. (What if another developer reads the code facing them?)
- Declaring a function with many parameters seems uncommon - especially when it contains only a few lines of code.

**Comment by NP199** [comment]
the std_img lib should be included via submodule as well.

File: src/menuscpp/SideBarMenu.cpp [hunk 1]
```diff
@@ -300,22 +243,22 @@ void SetSideBarMenu(
    // ImGui::PushStyleColor(
```
[the remark this answers is not in the document]
Comment (AKMaily) [reply src/menuscpp/SideBarMenu.cpp:243]: Yes they are, those menus will be added later

**Pull request merged**